packages/contracts/contracts/MarketRegistry.sol

```diff
@@ -130,44 +133,47 @@ contract MarketRegistry is
     /**
      * @notice Creates a new market.
      * @param _initialOwner Address who will initially own the market.
-     * @param _paymentCycleDuration Length of time in seconds before a bid's next payment is required to be made.
+     * @param _paymentCycleValue Length of time in seconds before a bid's next payment is required to be made.
      * @param _paymentDefaultDuration Length of time in seconds before a loan is considered in default for non-payment.
      * @param _bidExpirationTime Length of time in seconds before pending bids expire.
      * @param _requireLenderAttestation Boolean that indicates if lenders require attestation to join market.
      * @param _requireBorrowerAttestation Boolean that indicates if borrowers require attestation to join market.
      * @param _paymentType The payment type for loans in the market.
      * @param _uri URI string to get metadata details about the market.
+     * @param _paymentCycleType The payment cycle type for loans in the market - Seconds or Monthly
      * @return marketId_ The market ID of the newly created market.
      */
     function createMarket(
         address _initialOwner,
-        uint32 _paymentCycleDuration,
+        uint32 _paymentCycleValue,
         uint32 _paymentDefaultDuration,
         uint32 _bidExpirationTime,
         uint16 _feePercent,
         bool _requireLenderAttestation,
         bool _requireBorrowerAttestation,
         V2Calculations.PaymentType _paymentType,
-        string calldata _uri
+        string calldata _uri,
+        PaymentCycleType _paymentCycleType
```

Comment:
`PaymentCycleType` will be stored as a `uint8` since its an enum - lets optimize the positioning of the variable to be somewhere _**before**_ `_uri` 

this is because type `string` is actually compiled to `bytes[]` and the rest of the smaller variables will fit into a single slot